script/cmds.go

```
@@ -1232,3 +1226,84 @@ func Break() Cmd {
 		},
 	)
 }
+
+func emptyFlags(fs *pflag.FlagSet) {
+	fs.BoolP("quiet", "q", false, "Suppress printing of non-empty output diff")
+	fs.BoolP("trim", "t", false, "Trim newline chars")
+}
+
+// Empty checks whether contents of a file, or contents of either "stdout" or "stderr"
+// are empty.
+// Returns non-nil error if this assertion fails.
+func Empty() Cmd {
+	return Command(
+		CmdUsage{
+			Args:    "file",
+			Summary: "check if file is empty",
+			Flags:   emptyFlags,
+			Detail: []string{
+				"The command succeeds if the file is empty.",
+				"File can be 'stdout' or 'stderr' to compare the stdout or stderr buffer from the most recent command.",
+			},
+		},
+		func(s *State, args ...string) (WaitFunc, error) {
+			return nil, doEmpty(s, args...)
+		})
+}
+
+func doEmpty(s *State, args ...string) error {
+	quiet, err := s.Flags.GetBool("quiet")
+	if err != nil {
+		return err
+	}
+	trim, err := s.Flags.GetBool("trim")
+	if err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return ErrUsage
+	}
+
+	name := args[0]
+	text, err := getActualFileData(s, name)
```

Shouldn't we return the error here?

---

yup fixed